doc/library/config.txt

```diff
@@ -728,6 +728,23 @@ import theano and print the config variable, as in:
 
     Location of the magma library.
 
+.. attribute:: config.ctc.enabled
+
+    Default: ``'False'``
+
+    If ``'True'``, use `warp-ctc <https://github.com/baidu-research/warp-ctc>`_ for CTC
+    loss computations.
+
+    If ``'False'``, disable warp-ctc.
```

Comment:
Currrently, it is useful to set `ctc.enabled` to `False` (done by default) to avoid jenking failures, as CTC is not yet installed on Jenkins machines.

Or, should we introduce a function like `ctc_available()` (like `dnn_available()`) to replace this configuration ?